aria_plugin/exceptions.py

```diff
@@ -20,6 +20,10 @@ class MissingPluginsException(NonRecoverableError):
     pass
 
 
+class PluginsAlreadyExistException(Exception):
```

Comment:
BaseException

Reply:
Why are we inheriting from `BaseException` here? That is almost never the correct thing to do (it means anyone wanting to wrap a block of code in a log & retry loop won't catch this exception, unless they also catch `BaseException`, which is a bad idea because `BaseException` includes interrupts from the system)